dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeFactoryImpl.java

```diff
@@ -526,12 +507,12 @@ private boolean dbDelete(ContentType type) throws DotDataException {
     	orderBy = "mod_date";
     }
     DotConnect dc = new DotConnect();
-    dc.setSQL(String.format(this.contentTypeSql.SELECT_QUERY_CONDITION, searchCondition.condition,orderBy));
+    dc.setSQL( String.format( this.contentTypeSql.SELECT_QUERY_CONDITION, SQLUtil.sanitizeCondition( searchCondition.condition ), orderBy ) );
     dc.setMaxRows(limit);
     dc.setStartRow(offset);
-    dc.addParam(searchCondition.search);
-    dc.addParam(searchCondition.search);
-    dc.addParam(searchCondition.search);
+    dc.addParam( SQLUtil.sanitizeParameter( searchCondition.search ) );
```

Comment:
We should not need to sanitize a parameter in a prepared statement, the drive should take care of this for us, right?